criu/util.c

```diff
@@ -652,8 +652,8 @@ int cr_daemon(int nochdir, int noclose, int *keep_fd, int close_fd)
 		if (close_fd != -1)
 			close(close_fd);
 
-		if ((*keep_fd != -1) && (*keep_fd != 3)) {
-			fd = dup2(*keep_fd, 3);
```

Comment:
The first question what we need to ask our self is why we added this dup2.  In this code, we set 0, 1, 2 to /dev/null, but keep_fd can be one of this file descriptors.

Reply:
that seems to be ancient history
@xemul, any comments?